app/controllers/spree/orders_controller_decorator.rb

```diff
@@ -28,7 +28,7 @@ def edit
       associate_user
 
       if @order.insufficient_stock_lines.present?
-        flash[:error] = t(:spree_inventory_error_flash_for_insufficient_quantity)
+        flash[:error] = Spree.t(:inventory_error_flash_for_insufficient_quantity)
```

Comment:
see https://github.com/openfoodfoundation/openfoodnetwork/pull/3095
you can keep t(:spree_inventory_error_flash_for_insufficient_quantity) and move the translation to our en.yml under spree: